contents/handbook/product/user-feedback.md

```diff
@@ -29,8 +33,8 @@ Ways to invite users for an interview:
 
 **After the call**
 1. Add the notes to the [Google Doc][feedback-doc], linking the recording if you did one.
-2. Share a short summary of the user interview in the `#posthog-feedback` Slack channel
-3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.
+2. Share a short summary of the user interview in the `#posthog-feedback` Slack channel.
+3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Person profile in case Engineering needs more context when scoping/building.
```

Comment:
```suggestion
3. If the user reported specific bugs or requested specific features, open the relevant issues in GitHub. Be sure to link to their person profile in case our engineers needs more context when scoping/building.
```